client/src/features/location-rating/location-rating.actions.js

```diff
@@ -9,3 +10,39 @@ export function updateLocationRatingAction(location, feature) {
     }
   };
 }
+
+export function updateSuccess(bool) {
+  return {
+    type: LOCATION_RATING_TYPES.saveDbSuccess,
+    payload: {
+      bool
+    }
+  };
+}
+
+export const saveLocationToDb = (name, lat, lon, features) => {
+  return dispatch => {
+    apiService
+      .post(`/someModels/${name}`, {
+        name,
+        lat,
+        lon,
+        rating: {
+          accessible_parking: features.includes('Accessible parking'),
+          front_door_ramp: features.includes('Ramp leading to the front door'),
+          automatic_front_door: features.includes(
+            'Front door opens automatically'
+          ),
```

Comment:
this seems to have more spacing than the other rate points? 
Just a minor detail, that doesn't affect the code